package.json

```diff
@@ -46,16 +49,26 @@
 	],
 	"devDependencies": {
 		"@microsoft/api-extractor": "7.40.1",
+		"@types/fs-extra": "11.0.3",
 		"@types/node": "20.11.17",
 		"@typescript-eslint/eslint-plugin": "7.0.1",
 		"@typescript-eslint/parser": "7.0.1",
 		"eslint": "8.56.0",
 		"eslint-plugin-tsdoc": "0.2.17",
+		"fs-extra": "11.1.1",		
```

Comment:
I don't see `fs-extra` being used anywhere, so these dependencies should be removed.